scripts/deno/ophan-components.ts

```diff
@@ -48,7 +56,7 @@ const getOphanComponents = (
 	const doc = new DOMParser().parseFromString(html, 'text/html');
 	if (!doc) throw new Error('Unable to parse DOM');
 
-	return [...doc.querySelectorAll(`[${attribute}]`)].filter(
+	return [...doc.querySelectorAll(`[${attribute}]:not(.is-hidden,.is-hidden *)`)].filter(
```

Comment:
👏 This removes tags that either
- have the `is-hidden` class `:not(.is-hidden)`
- are a descendent of an item that has the `is-hidden` class `:not(.is-hidden *)`

This is required as these elements are not displayed and therefore unclickable.